indico/modules/events/blueprint.py

```diff
@@ -37,3 +38,7 @@
 
 _bp.add_url_rule('/event/<confId>/manage/external-ids', 'manage_event_references', RHManageReferences,
                  methods=('GET', 'POST'))
+_bp.add_url_rule('/event/<confId>/manage/location', 'manage_event_location', RHManageEventLocation,
+                 methods=('GET', 'POST'))
+_bp.add_url_rule('/event/<confId>/manage/people', 'manage_event_person_links', RHManageEventPersonLinks,
```

Comment:
not a good url for it.. this could be a nice url for the future "list of persons in the event" page.

maybe `/manage/event-location` and `/manage/event-persons`? those are rather unlikely to be used for those pages